recipes/mpich/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# FIXME: This is a terrible workaround.
+# Ideally we should fix the information the .la files.
+rm -rf $PREFIX/lib/libquadmath.la
+rm -rf $PREFIX/lib/libgfortran.la
```

Comment:
I believe we can avoid this is we don't need the static libs (`--enable-static=no`). @bekozi are they essential?